migrations/00011_hashed_refresh_tokens.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "auth"."refresh_tokens"
+    ALTER COLUMN "refresh_token" SET DATA TYPE Text;
```

Comment:
This is a breaking change because the data type of the column changes from `uuid` to `text`.